packages/frontend/src/components/AppPicker/index.tsx

```diff
@@ -235,7 +234,7 @@ export function AppPicker({ className, onSelect, apps = [] }: Props) {
   }
 
   return (
-    <div className={classNames(styles.appPickerContainer, className)}>
```

Comment:
Note that this was actually a bug, since `styles.appPickerContainer` here doesn't exist.